noetic/distribution.yaml

```
@@ -6602,6 +6602,27 @@ repositories:
       url: https://github.com/ROBOTIS-GIT/robotis_manipulator.git
       version: noetic-devel
     status: maintained
+  rom_msgs:
+    doc:
+      depends:
+      - actionlib
+      - robot_state_publisher
+      - std_msgs
+      type: git
+      url: https://github.com/greenghostman/rom_msgs.git
+      version: master
```

```suggestion
      version: main
```

I don't see a `master` branch on the `rom_msg` repo; I think this should be [`main`](https://github.com/GreenGhostMan/rom_msgs/branches).